### ISSUE TITLE
`ignore_startup_parameters` for pgbouncer ini not working still

### ISSUE THREAD

**Issue by redscaresu** [zalando/postgres-operator#1029]
hi, my apps are failing to connect to the pooler due to extra_float_digits.

This should have been fixed by https://github.com/zalando/postgres-operator/issues/984 however it does not seem to work.

According to @ilya-lesikov this is due to "extra_float_digits, options" being wrapped in quotes.  Very happy to test if you can release a new image that addresses this issue.

Thanks for your time.

**Comment by FxKu** [comment]
@redscaresu please use the latest image: `registry.opensource.zalan.do/acid/postgres-operator:v1.5.0-14-g0c6655a-dirty`. We don't overwrite release images.

**Comment by ilya-lesikov** [comment]
@FxKu not sure if this `registry.opensource.zalan.do/acid/postgres-operator:v1.5.0-14-g0c6655a-dirty` image should change anything, but I see that `ignore_startup_parameters = "extra_float_digits, options"` is hardcoded into `master-8` pgbouncer image.

Few days ago I used one of the latest releases of the the operator with `master-8` pgbouncer image. It worked only after I exec'd into the pgbouncer pods, changed `ignore_startup_parameters = "extra_float_digits, options"` to `ignore_startup_parameters = extra_float_digits,options` in pgbouncer.ini and executed a pgbouncer RELOAD command, which reloads the pgbouncer config file without restarting the pgbouncer itself.

After that `SHOW CONFIG` pgbouncer command showed me `extra_float_digits,options` instead of `"extra_float_digits, options"` as a runtime `ignore_startup_parameters` value, and then the pgbouncer started serving the requests, and did that until I restarted the pod.

Thanks.
